composer.json

```diff
@@ -9,26 +9,26 @@
     "require": {
         "php": ">=5.5",
         "ext-curl": "*",
-        "guzzlehttp/guzzle": "~4.0",
```

Comment:
it is not so much if it follows or not. Is more about whether the maintainer knows exactly what/how its dependencies evolve.